src/python/views/demo_bio_01_funcoup/__init__.py

```diff
@@ -0,0 +1,307 @@
+        'D.melanogaster': 'D.melanogaster',
+        'D.rerio': 'D.rerio',
+        'E.coli': 'E.coli',
+        'G.gallus': 'G.gallus',
+        'H.sapiens': 'H.sapiens',
+        'M.jannaschii': 'M.jannaschii',
+        'M.musculus': 'M.musculus',
+        'O.sativa': 'O.sativa',
+        'P.falciparum': 'P.falciparum',
+        'R.norvegicus': 'R.norvegicus',
+        'S.cerevisiae': 'S.cerevisiae',
+        'S.pombe': 'S.pombe',
+        'S.scrofa': 'S.scrofa',
+        'S.solfataricus': 'S.solfataricus',
+    }
+
+    base_url = os.environ['BASE_URL']
+
+    filter_by_org_type_init = urlParams.get_field('filter_by_org', default='B.subtilis')
+    filter_by_org_type = \
+        st.sidebar.selectbox(
+            'Choose organism:',
+            ('A.thaliana', 'B.subtilis', 'B.taurus','C.elegans','C.familiaris','C.intestinalis','D.discoideum','D.melanogaster','D.rerio','E.coli','G.gallus','H.sapiens','M.jannaschii','M.musculus','O.sativa','P.falciparum','R.norvegicus','S.cerevisiae','S.pombe','S.scrofa','S.solfataricus'),
+            index=('A.thaliana', 'B.subtilis', 'B.taurus','C.elegans','C.familiaris','C.intestinalis','D.discoideum','D.melanogaster','D.rerio','E.coli','G.gallus','H.sapiens','M.jannaschii','M.musculus','O.sativa','P.falciparum','R.norvegicus','S.cerevisiae','S.pombe','S.scrofa','S.solfataricus').index(filter_by_org_type_init),
+            format_func=(lambda option: species_to_label[option]))
+    urlParams.set_field('filter_by_org', filter_by_org_type)
+
+
+    st.sidebar.title("Network Type")
+    net_to_label = {
+        'compact': 'compact',
+        'full': 'full',
+    }
+
+    filter_by_net_type_init = urlParams.get_field('filter_by_net', default='full')
+    filter_by_net_type = \
+        st.sidebar.selectbox(
+            'Select FunCoup Network:',
+            ('compact','full'),
+            index=('compact','full').index(filter_by_net_type_init),
+            format_func=(lambda option: net_to_label[option]))
+    urlParams.set_field('filter_by_net', filter_by_net_type)
+
+
+    st.sidebar.title("Choose Edge Weight")
+    edge_to_label = {'PFC':'PFC', 'FBS_max':'FBS_max', 'FBS_PPI':'FBS_PPI',
+       'FBS_Metabolic':'FBS_Metabolic', 'FBS_Signaling':'FBS_Signaling', 'LLR_MEX':'LLR_MEX', 'LLR_PIN':'LLR_PIN',
+       'LLR_SCL':'LLR_SCL', 'LLR_PHP':'LLR_PHP', 'LLR_MIR':'LLR_MIR', 'LLR_TFB':'LLR_TFB', 'LLR_PEX':'LLR_PEX',
+       'LLR_GIN':'LLR_GIN', 'LLR_DOM':'LLR_DOM', 'LLR_QMS':'LLR_QMS', 'LLR_HSA':'LLR_HSA', 'LLR_MMU':'LLR_MMU',
+       'LLR_RNO':'LLR_RNO', 'LLR_CFA':'LLR_CFA', 'LLR_GGA':'LLR_GGA', 'LLR_DRE':'LLR_DRE', 'LLR_CIN':'LLR_CIN',
+       'LLR_DME':'LLR_DME', 'LLR_CEL':'LLR_CEL', 'LLR_BTA':'LLR_BTA', 'LLR_SCE':'LLR_SCE', 'LLR_ATH':'LLR_ATH',
+       'LLR_SPO':'LLR_SPO', 'LLR_BSU':'LLR_BSU', 'LLR_ECO':'LLR_ECO', 'LLR_PFA':'LLR_PFA', 'LLR_DDI':'LLR_DDI',
+       'LLR_SSO':'LLR_SSO', 'LLR_SSC':'LLR_SSC', 'LLR_MJA':'LLR_MJA', 'LLR_OSA':'LLR_OSA','Max_category':'Max_category'
```

Comment:
can these sorts of menus be dynamically populated by crunching over CSV fields?

Reply:
just made the overall interface more logical so no need for these fields anymore